app/views/spree/products/show.html.erb

```diff
@@ -1,51 +1,359 @@
       <div class="col-12 col-md-7 col-lg-6">
-        <%= render partial: 'description' %>
+        <%#= render partial: 'description' %>
       </div>
       <div class="d-none d-lg-block col-lg-1"></div>
       <div class="col-12 col-md-5" data-hook="product_properties">
-        <%= render partial: 'properties' %>
+        <%#= render partial: 'properties' %>
       </div>
       <div class="col-12" data-hook="promotions">
-        <%= render partial: 'promotions' %>
+        <%#= render partial: 'promotions' %>
       </div>
     </div>
 
     <div id="no-product-available" class="no-product-available-dropdown">
-      <%= render partial: 'spree/shared/no_product_available' %>
+      <%#= render partial: 'spree/shared/no_product_available' %>
     </div>
 
-    <%= render partial: 'gallery_modal' %>
+    <%#= render partial: 'gallery_modal' %>
   </div>
-  <div id="related-products"></div>
-
-  <%= products_structured_data([@product]) %>
+  <div id="related-products"></div> -->
 
-  <%= render 'spree/shared/product_added_modal' %>
-<% end %>
+  <%#= products_structured_data([@product]) %>
 
+  <%#= render 'spree/shared/product_added_modal' %>
+<%# end %>
+<% @body_id = 'product-details' %>
 <div
   data-related-products
   data-related-products-id="<%= @product.slug %>"
   data-related-products-enabled="<%= @product.respond_to?(:has_related_products?) %>" />
+  <main class="page-contents">
+    <div class="row no-gutters">
+      <div class="col-xl-1 tablet-hide"></div>
+      <div class="col-xl-8 col-lg-8">
+        <div class="product-detail-wrapper">
+            <%= spree_breadcrumbs(@taxon, '', @product) %>
+          <div class="title-wrap position-relative">
+            <h1 class="title"> <%= @product.name %></h1>
+            <div class="ratings">  <%= image_tag "star.svg", alt:'star' %> <%= image_tag "star.svg", alt:'star' %> <%= image_tag "star.svg", alt:'star' %> <%= image_tag "star.svg", alt:'star' %> <%= image_tag "star-off.svg", alt:'star' %><span>5 Reviews</span></div>
+            <ul class="product-standards d-flex flex-wrap align-items-center phone-visible">
+              <li class="glutenfree-icon" data-toggle="tooltip" data-placement="bottom" title="Gluten free">Gluten free</li>
+              <li class="nongmo-icon" data-toggle="tooltip" data-placement="bottom" title="Non GMO">Non GMO</li>
+              <li class="u-icon" data-toggle="tooltip" data-placement="bottom" title="U">u</li>
+              <li class="vegan-icon" data-toggle="tooltip" data-placement="bottom" title="Vegan">Vegan</li>
+            </ul>
+          </div>
+          <div class="product-desc-wrap" data-hook="product_right_part"> 
+            <div class="row" data-hook="product_show">
+              <%= render partial: 'gallery_modal' %>
+              <div class="col-xl-5">
+                <%= render partial: 'phone_cart' %>
+                <div  id="product-description" data-hook="product_right_part_wrap">
```

Comment:
datahook "product_right_part" is not in right place 
cart and description both partials are in this hook